css/style.css

```diff
@@ -0,0 +1,586 @@
+  height: 4px;
+  margin: 25px auto;
+  top: 100%;
+  background-color: var(--text-orange);
+  border: 1px solid var(--text-orange);
+  border-radius: 24px;
+  display: block;
+  content: '';
+}
+
+.classes-list {
+  display: flex;
+  flex-direction: column;
+  margin: 5% auto;
+  padding: 20px;
+}
+
+.class-items {
+  display: flex;
+  flex-direction: row;
+  padding: 0;
+  height: 20vh;
+  margin: 0.3%;
+  background: rgba(233, 233, 233, 0.1);
+  box-shadow: 0 3px 10px rgb(0 0 0 / 50%);
+  align-items: center;
+}
+
+.class-items img {
+  margin: 0 4%;
+  width: 10%;
+  background: rgba(255, 255, 255, 0.49);
+  border: 1px solid var(--black);
+  border-radius: 8px;
+  padding: 3px;
+}
+
+.class-items h3 {
+  color: var(--text-orange);
+  margin: 0;
+  font-size: 16px;
+}
+
+.class-items p {
+  text-align: start;
+  margin: 0 3%;
+  font-family: cocogoose, sans-serif;
+  font-size: 10px;
+  color: var(--texts);
+}
+
+#see-program-button {
+  padding: 20px 20px 20px;
+  background: var(--text-orange);
+  color: var(--texts);
+  font-size: 17px;
+  font-weight: 500;
+  border: none;
+  border-radius: 2px;
+  margin: 10px auto;
+  width: 75%;
+}
+
+.two-column-grid {
+  display: grid;
+  grid-template-columns: auto;
+  grid-template-rows: 150px 150px 150px 150px 150px 150px;
+  grid-gap: 15px 15px;
+  margin: 15px 5%;
+  text-align: start;
+  align-items: center;
+}
+
+.grid-items {
+  display: flex;
+  background: url(../assets/background/trainer-bg.png);
+  background-position: top left;
+  background-repeat: no-repeat;
+}
+
+.grid-items img {
+  margin-top: 3%;
+  margin-left: 3%;
+  width: 100px;
+  height: 100px;
+}
+
+.trainer-desc {
+  margin-left: 10px;
+}
+
+.trainer-desc h3 {
+  margin: 0;
+  padding: 0;
+  font-size: 16px;
+}
+
+.trainer-qualification {
+  color: var(--text-orange);
+  font-size: 14px;
+  font-weight: 600;
+  font-style: italic;
+  margin: 5px 0;
+}
+
+.trainer-desc hr {
+  width: 3%;
+  border-color: var(--text-orange);
+  margin: 5px 0;
+}
+
+.about-trainer {
+  font-weight: bold;
+  font-size: 11px;
+  margin: 15px 0;
+}
+
+.partner-section {
+  background: #272a31;
+  padding: 15px;
+}
+
+.partner-list {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-wrap: wrap;
+}
+
+.partner-list-items {
+  flex: 0 1 85px;
+  margin: 10px 15px;
+}
+
+.partner-list-items img {
+  width: 85px;
+}
+
+.sessions-main-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.session-container-items {
+  flex: 0 1 auto;
+  margin: 15px;
+  height: 100%;
+}
+
+.session-one {
+  background: url(../assets/images/sessions/session-one.jpg);
+  background-size: cover;
+}
+
+.session-two {
+  background: url(../assets/images/sessions/session-two.jpg);
+  background-size: cover;
+}
+
+.session-text-holder {
+  background: rgba(248, 8, 8, 0.5);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding: 15px;
+}
+
+footer {
```

Comment:
- [ ] Please increase a little bit of white color in your **mobile** footer because the background of the design is a white color.
- [ ] Desktop Size:
- Please "About Page" has a dark background in the footer, ([See project](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015))